dash/dash.py

```diff
@@ -1333,7 +1333,11 @@ def dispatch(self):
         except KeyError as missing_callback_function:
             msg = "Callback function not found for output '{}', perhaps you forgot to prepend the '@'?"
             raise KeyError(msg.format(output)) from missing_callback_function
-        response.set_data(func(*args, outputs_list=outputs_list))
+
+        result_data = DashSerializer.serialize_tree(
```

Comment:
I'm not sure we need this here, I think at this point, the data should already be serialized?